test/common/common/version_test.cc

```diff
@@ -34,6 +35,7 @@ TEST(VersionTest, BuildVersion) {
             fields.at(BuildVersionMetadataKeys::get().RevisionStatus).string_value());
   EXPECT_EQ(VersionInfoTestPeer::buildType(),
             fields.at(BuildVersionMetadataKeys::get().BuildType).string_value());
+  EXPECT_FALSE(VersionInfoTestPeer::sslFipsCompliant());
```

Comment:
This will fail tests if build is FIPS compliant.

Reply:
@PiotrSikora Is there a way I can make this conditional? I wanted to add a test for validating fips as well as non-fips, but I couldn't find a way to determine that. Seems like currently all builds are running in non-fips mode.

Reply:
`compile_time_options` uses `--define boringssl=fips`, and it passed, so something is broken.